docs/security/azure-linux-os-guard.md

```diff
@@ -0,0 +1,583 @@
+|--------------|-----------------------------------------------------------------------------------|
+| Permissive   | Logs policy violations but does not enforce restrictions. Helps identify potential issues without impacting workloads. |
+| Enforcing    | Applies mandatory access control policies, blocking actions that violate SELinux rules. Provides strong runtime containment. |
+| Disabled     | SELinux is turned off; no access control or logging occurs. Not recommended for hardened environments. |
+
+*Please note, for Azure Linux with OS Guard Public Preview SELinux is in Permissive mode, with plans to move to Enforcing mode in GA.*
+
+Together, IPE and SELinux provide defense in depth: preventing execution of tampered or unauthorized code, blocking privilege escalation, and maintaining strong isolation between workloads.
+
+To begin exploring these security modules in more depth, ensure you still have access to the node from Scenario 2 through a privileged container as a debugging pod and can interact with the node session by running `chroot /host`. 
+
+You will first verify that SELinux is in `permissive` mode on your node. You can do so by running the following command:
+
+```bash
+getenforce
+```
+If you prefer to see a more detailed summary of the SELinux status with information on the policy type, you can do so by running the following command:
+
+```bash
+sestatus
+```
+
+You will now verify that IPE is in `audit` mode on your node. 
+
+Let's start by checking if IPE was enabled at boot time by running the following command:
+```bash 
+ cat /proc/cmdline
+ ```
+You should see `ipe.enforce=0` in the output, indicating that IPE is enabled in audit mode. *Note: this command can also be used to verify if SELinux was enabled in permissive mode at boot time by looking for `selinux=1` in the output.*
+
+You can also check the current IPE mode by running the following command. *Note: an output of 0 indicates that IPE is in audit mode.*:
+```bash
+cat /sys/kernel/security/ipe/enforce
+```
+Let's now explore an example of how SELinux and IPE work in tandem to prevent the execution of untrusted binaries and confine processes access to strict security domains. 
+
+You will begin by executing a trusted, signed binary from the Azure Linux OS Guard /usr directory. To do so, run the following command:
+
+```bash
+/usr/bin/true && echo "Binary has executed"
+```
+Now let's inspect journalctl to see if SELinux or IPE logged any events following the execution of `/usr/bin/true`.
+
+First, inspect the SELinux logs by running the following command:
+```bash
+journalctl -g 'AVC.*path="/usr/bin/true"' | tail -n 30
+```
+You should see the following output:
+```
+-- No entries --
+```
+Since /usr/bin/true is a trusted binary in /usr (protected by dm-verity), there’s no violation to log. AVC logs appear only when an action violates SELinux policy. For example, if a process tries to access a restricted resource or execute an untrusted binary.
```

Comment:
```suggestion
Since `/usr/bin/true` is a trusted binary in `/usr` (protected by `dm-verity`), there’s no violation to log. AVC logs appear only when an action violates SELinux policy. For example, if a process tries to access a restricted resource or execute an untrusted binary.
```